### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0xParth**
Added DNS details

### CODE REVIEW

**Review by ppsyllos33 — COMMENTED** [review]
Looks Good

**Comment by realArcherL** [comment]
Hey @0xParth good work resolving the conflicts, but I think after the merge, the main branch has overwritten your changes. Please update the branch and will merge it

**Review by 0xParth — COMMENTED** [review]
updates

**Comment by 0xParth** [comment]
This one

File: README.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ pyOpenSSL: v23.0.0
 
 # The versions can change depending on the time of implementation
 ```
+```
 Custom Domain: 0xparthhackerone.me ( Hosting: Namecheap.me)
 Public Facing DNS Server: 34.121.172.3
```
Comment (realArcherL) [inline README.md]: Please remove the IP address from here. We are not adding it. Also, these are ephemeral, so there is no point sharing this.

**Review by realArcherL — CHANGES_REQUESTED** [review]

**Pull request closed without merging**